private/buf/cmd/buf/buf.go

```diff
@@ -146,6 +146,14 @@ func NewRootCommand(name string) *appcmd.Command {
 					stats.NewCommand("stats", builder),
 					migratev1beta1.NewCommand("migrate-v1beta1", builder),
 					studioagent.NewCommand("studio-agent", noTimeoutBuilder),
+					{
+						Use:   "plugin",
+						Short: "Manage plugins on the Buf Schema Registry",
+						SubCommands: []*appcmd.Command{
+							pluginpush.NewCommand("push", builder),
+							curatedplugindelete.NewCommand("delete", builder),
```

Comment:
```suggestion
							plugindelete.NewCommand("delete", builder),
```

Maybe as part of this we remove the old `buf beta registry plugin` and `buf beta registry template` commands since the old RPCs no longer support making changes?